packages/wonder-blocks-button/components/button.stories.js

```diff
@@ -0,0 +1,121 @@
+
+import Button from "./button.js";
+
+export default {title: "Button"};
+
+export const basicButtons = () => (
+    <View>
+        <View style={{flexDirection: "row"}}>
+            <Button>Hello, world!</Button>
+            <Strut size={16} />
+            <Button kind="secondary">Hello, world!</Button>
+            <Strut size={16} />
+            <Button kind="tertiary">Hello, world!</Button>
+        </View>
+        <Strut size={16} />
+        <View style={{flexDirection: "row"}}>
+            <Button disabled={true}>Hello, world!</Button>
+            <Strut size={16} />
+            <Button disabled={true} kind="secondary">
+                Hello, world!
+            </Button>
+            <Strut size={16} />
+            <Button disabled={true} kind="tertiary">
+                Hello, world!
+            </Button>
+        </View>
+        <Strut size={16} />
+        <View style={{flexDirection: "row"}}>
+            <Button color="destructive">Hello, world!</Button>
+            <Strut size={16} />
+            <Button kind="secondary" color="destructive">
+                Hello, world!
+            </Button>
+            <Strut size={16} />
+            <Button kind="tertiary" color="destructive">
+                Hello, world!
+            </Button>
+        </View>
+    </View>
+);
+
+export const darkBackgroundButtons = () => (
+    <View>
+        <View style={{flexDirection: "row"}}>
+            <Button light={true}>Hello, world!</Button>
+            <Strut size={16} />
+            <Button light={true} kind="secondary">
+                Hello, world!
+            </Button>
+            <Strut size={16} />
+            <Button light={true} kind="tertiary">
+                Hello, world!
+            </Button>
+        </View>
+        <Strut size={16} />
+        <View style={{flexDirection: "row"}}>
+            <Button light={true} disabled={true}>
+                Hello, world!
+            </Button>
+            <Strut size={16} />
+            <Button light={true} disabled={true} kind="secondary">
+                Hello, world!
+            </Button>
+            <Strut size={16} />
+            <Button light={true} disabled={true} kind="tertiary">
+                Hello, world!
+            </Button>
+        </View>
+        <Strut size={16} />
+        <View style={{flexDirection: "row"}}>
+            <Button light={true} color="destructive">
+                Hello, world!
+            </Button>
+            <Strut size={16} />
+            <Button light={true} kind="secondary" color="destructive">
+                Hello, world!
+            </Button>
+            <Strut size={16} />
+            <Button light={true} kind="tertiary" color="destructive">
+                Hello, world!
+            </Button>
+        </View>
+    </View>
+);
+
+darkBackgroundButtons.story = {
+    parameters: {
+        backgrounds: [{name: "red", value: Color.darkBlue, default: true}],
```

Comment:
I'm curious, why the name "red"? Does Storybook require a key name for that object?

Reply:
I copy/pasted this code from an example without changing all of the things I needed to.